### PR TITLE
raidboss: add Lost City of Amdapor Hard

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        return { alertText: output.pop!() };
+      },
+    },
+    {
+      id: 'LostCityHard Winged Lion Ancient Aero',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '15CE', source: 'Winged Lion', capture: false }),
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
```
Comment (JLGarber) [inline ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts:147]: It's probably okay, but the first time through I was slightly confused by this being a `startsUsing` trigger as opposed to `ability`, since the orb doesn't appear until the castbar completes.

File: ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    },
+    {
+      id: 'LostCityHard Achamoth Toxic Squama',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0001' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Away from orb',
```
Comment (JLGarber) [inline ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts:56]: Would it be possible to reword this as "Avoid chasing orb" or something similar, and call it for everyone? It shouldn't be too noisy in this encounter.

**Review by JLGarber — APPROVED** [review]
The Mana Pot trash before boss 3 uses Mysterious Light, `16C8`, a gaze attack. Is this worth adding a trigger for?

**Review by quisquous — COMMENTED** [review]
> The Mana Pot trash before boss 3 uses Mysterious Light, 16C8, a gaze attack. Is this worth adding a trigger for?

Done, good idea, thanks!
